assets/js/WfModule.test.js

```diff
@@ -1,41 +1,39 @@
 import React from 'react'
-// *** Breaks when importing: Unsolved Mysteries! ****
-// import WfModule from './WfModule'
+import { WfModule } from './WfModule'
 import { shallow } from 'enzyme'
 
 describe('WfModule, NOT read-only mode - DUMMY TEST ONLY', () => {
 
   var wrapper;
   var props = {
         'data-isReadOnly': false,
-        'data-wfmodule': {},
+        'data-wfmodule': {
+          'notes': [],
+          'parameter_vals': [],
+          'module_version': {
+            'module': {},
+          }
+        },
         'data-changeParam': () => {} ,
         'data-removeModule':  () => {} ,
         'data-revision': 707,
         'data-selected': false,
         'data-api': {},
+        'connectDragSource': (component) => {return component},
+        'connectDropTarget': (component) => {return component},
         'focusModule': () => {}
       };
 
-  // beforeEach(() => {
-  //   wrapper = shallow(
-  //     <WfModule
-  //       {...props}
-  //   />)
-  // });
+  beforeEach(() => {
+    wrapper = shallow(
+      <WfModule
+        {...props}
+      />
+    )
+  });
 
   it('Renders - dummy test only', () => {
-
-    expect(true).toBe(true);
-
-    // expect(wrapper).toMatchSnapshot();
-
-
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Set focusModule to jest.fn(), use mount() to get lifecycle methods called, check that setFocus was called once